filters/renderHTMLCorrectly.js

```diff
@@ -0,0 +1,8 @@
+(function () {
+	angular.module('app')
+		.filter('renderHTMLCorrectly', function ($sce)	{
```

Comment:
This got broken on minify. Always you inject something, do it using $inject, or better, array notation. In this case, this line would better be:

```javascript
.filter('renderHTMLCorrectly', ['$sce', renderHTMLCorrectly]);

function renderHTMLCorrectly($sce) {
    // ... 
```

Also, the name is not accurate, because HTML always renders correctly, since that's a browser job. This doesn't render but trust HTML. Maybe a name as `trustHtml` would be better. Be careful with this kind of things, since it might make us vulnerable to XSS Attacks.

Reply:
The point of this is just to include the filter where it's needed. In this case, it's included in a no input field

Reply:
Except if in some moment part of the error definition came dynamically from the server (i.e.). Anyway, that's not so much clean, adding nodes as text, so I've changed error from string to an array, so it can be shown in a ng-repeat, letting angular manage all the view.

b9414d4